src/main/java/mate/project/exception/CustomGlobalExceptionHandler.java

```diff
@@ -0,0 +1,59 @@
+package mate.project.exception;
+
+import java.time.LocalDate;
```

Comment:
Consider using java.time.ZonedDateTime or java.time.Instant instead of LocalDate in order to include the time and timezone information along with the date.